src/bsp/platform/iotc_bsp_io_net_test.cc

```diff
@@ -76,28 +77,32 @@ bool ServerTest::WaitUntilSocketReadyForRead() {
 }
 
 TEST_P(ServerTest, EndToEndCommunicationWorks) {
-  const NetworkType* test_case = GetParam();
-  socket_type_ = test_case->socket_type;
-  listening_addr_ = test_case->listening_addr;
-  protocol_type_ = test_case->protocol_type;
+  const NetworkType test_case = GetParam();
+  const char* kTestMsg = "hello\n";
+  const uint16_t kTestPort = 2000;
+  std::string listening_addr_ = test_case.listening_addr;
```

Comment:
s/listening_addr_/listening_addr

Do not add the `_` suffix, if it's not a member variable.